kubectl-cloudflow/cmd/update_docker_registry_credentials.go

```diff
@@ -158,23 +165,22 @@ func createOrUpdateServiceAccount(k8sClient *kubernetes.Clientset, appID string,
 	return nil, fmt.Errorf("failed to update Cloudflow app service account in `%s`, %s", appID, err)
 }
 
-func newCloudflowServiceAccount(appID string, cloudflowOperatorOwnerReference metav1.OwnerReference) v1.ServiceAccount {
+func newCloudflowServiceAccount(appID string) v1.ServiceAccount {
```

Comment:
is this a piggy-back from the owner-ref issue?  It doesn't look like it's related to the status bug.

Reply:
yes I made this change on top of the owner-ref issue, strange that this pops up though... Let me double check.

Reply:
It's outdated, I'm doing this on top of the fix for owner-ref issue. (It is in no commit in this PR)